Update Docker

westim opened the pull request:
Docker recently announced the beta for Software & Hardware 2FA for the Docker Hub site.

https://www.docker.com/blog/designing-docker-hub-2fa/

westim committed:
Update Docker 2FA

Docker recently announced the beta for Software & Hardware 2FA for the Docker Hub site.

https://www.docker.com/blog/designing-docker-hub-2fa/

Carlgo11 commented on _data/developer.yml:
```diff
@@ -193,6 +193,10 @@ websites:
     twitter: Docker
     facebook: docker.run
```
```suggestion
```

kmpoppe replied:
Resolved by 414add7

Carlgo11 commented on _data/developer.yml:
```diff
@@ -193,6 +193,10 @@ websites:
     twitter: Docker
```
```suggestion
```

kmpoppe replied:
Resolved by 414add7

Carlgo11 commented on _data/developer.yml:
```diff
@@ -193,6 +193,10 @@ websites:
     facebook: docker.run
     img: docker.png
+    tfa:
+      - software
```
```suggestion
      - totp
```

kmpoppe replied:
Resolved by 414add7

Carlgo11 commented on _data/developer.yml:
```diff
@@ -193,6 +193,10 @@ websites:
     img: docker.png
+    tfa:
+      - software
+      - hardware
```
By the looks of it they don't support WebAuthn or any other hardware solutions yet.
```suggestion
```

kmpoppe replied:
Resolved by 414add7

kmpoppe committed:
Applying Carlgo's suggestions

Resolves https://github.com/2factorauth/twofactorauth/pull/4165/files#r338156071
Resolves https://github.com/2factorauth/twofactorauth/pull/4165/files#r338156007
Resolves https://github.com/2factorauth/twofactorauth/pull/4165/files#r338156149
Resolves https://github.com/2factorauth/twofactorauth/pull/4165/files#r338157001
